tools/fast_pusher_.py

```diff
@@ -119,7 +126,7 @@ def main():
 
   with docker_session.Push(name, creds, transport, threads=_THREADS) as session:
     with v2_2_image.FromDisk(config, zip(args.digest or [], args.layer or []),
-                             legacy_base=args.tarball) as v2_2_img:
+                             legacy_base=legacy_tarball) as v2_2_img:
```

Comment:
Sorry, what's the bug here? Note that `client/v2_2/docker_image_.py` changes the type of `legacy_base` from a string to a tarball.